server/utils/utils.ts

```diff
@@ -109,14 +109,18 @@ export const convertToTitleCase = (sentence: string): string =>
 export const visitDateAndTime = ({
   startTimestamp,
   endTimestamp,
+  isIntTest = false,
```

Comment:
Made this an optional, allows for the integration tests to return a different format for checking as cypress removes HTML tags.